### PR TITLE
fixes #213 - created a slide in drawer for main page categories (mobile version)

### LINKED ISSUE #213: Provide a slide in drawer for categories in main page for mobile version
Presently on mobile version the left sidebar (categories) is present as a dropdown, however it is not accessible once the user scrolls down, provide a slide in menu drawer like the details page which can be accessed anytime.
Also the index.html file has lots of indentation problems which makes reading the code tough. It would be nice and less pain to read the code if it is properly indented.

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #213. Created a slide in drawer for categories
in start page for mobile version.
Indented and cleaned index.html

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link : loklakapps2.surge.sh

screenshot_2017-06-23-22-02-31-716_com android chrome
screenshot_2017-06-23-22-02-56-710_com android chrome
screenshot_2017-06-23-22-03-10-992_com android chrome
screenshot_2017-06-23-22-03-48-064_com android chrome

**Comment by djmgit** [comment]
@SKrPl @kavithaenair @Achint08 @hemantjadon open for review

**Comment by hemantjadon** [comment]
@djmgit Test link?

**Comment by singhpratyush** [comment]
@hemantjadon: http://loklakapps2.surge.sh

File: css/dashboard.css [hunk 1]
```diff
@@ -72,20 +56,38 @@ ul.request {
...
+  color: white;
+  background-color: #C8254D;
+  border-radius: 10%;
+  top: 30%;
+  left: 2%;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  cursor: pointer;
+  transition: all 0.5s;
+  z-index: 2;
+  transition: all 0.3s;
```
Comment (Achint08) [inline css/dashboard.css]: Use just one transition

Comment (hemantjadon) [inline css/dashboard.css]: @djmgit don't use `all` in transitions, use comma separated transitions.

**Comment by djmgit** [comment]
@hemantjadon @Achint08 made the changes

**Comment by singhpratyush** [comment]
@djmgit: Please avoid using `png` where `glyphicon` can be used. Also, two hamburger menus look a bit weird here. What do you think of this?
loklakapps2 surge sh- iphone 6 plus

* It is using `glyphicon` instead of image.
* It is not covering any app in the view.
* It is at the top of the side drawer instead of somewhere in between.

**Comment by djmgit** [comment]
@hemantjadon thanks a lot for the suggestions, I will implement them ASAP

File: css/dashboard.css [hunk 2]
```diff
@@ -72,20 +56,37 @@ ul.request {
...
+  padding: 12px;
+  color: white;
+  background-color: #C8254D;
+  border-radius: 10%;
+  top: 30%;
+  left: 2%;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  cursor: pointer;
+  z-index: 2;
+  transition: top 0.3s;
```
Comment (hemantjadon) [inline css/dashboard.css]: Nope, only stick to transiotion of `transform` and `opacity` as they don't trigger `Paint`.

**Comment by djmgit** [comment]
@singhpratyush @hemantjadon made the changes.

sh

**Comment by hemantjadon** [comment]
Okay one interesting thing I just noticed, is when you change the device width from small to large the drawer doesn't come up automatically, I wonder why it is so? Please change this behaviour.
Try resizing the window.

**Comment by djmgit** [comment]
Test link : http://loklakapps2.surge.sh/

@singhpratyush I have fixed the menu button ( at the position where it is shown in the previous screen shot ). Also I have removed the overlapping problem however I had to use !important to do it. I followed this article here https://coderwall.com/p/wpjw4w/change-the-bootstrap-navbar-breakpoint . It is a kind of hacky workaround since bootstrap3 does not provide any easy way to change the breakpoint where hamburger appears.

@hemantjadon when the screen gets smaller I am using javascript to control the sidebar. On mobile screen its not a problem (as there is no option of enlarging the screen) but on browser, if some one squeezes the screen, uses the sidebar, closes it and then again expands the screen then the sidebar is gone as js is overriding css media query which it will obviously do. I am trying to find a solution for it. I can either remove the js part and use css animation or I can remove the media query part and use js window resize listener. Which one do you suggest. Also do let me know if there is a better way to do it.
Also should I create a separate issue for it. As this PR has already become quite long.

**Comment by hemantjadon** [comment]
@djmgit Let's Iterate here not making this too Long.
Open up another issue. There are better approaches for sidenavs, I can code up some sample sidebar or some reference. Let's iterate here :) 👍

**Pull request merged**